packages/earn-controller/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Bump `@metamask/controller-utils` to `^11.9.0` ([#5812](https://github.com/MetaMask/core/pull/5812))
+- **BREAKING:** bump `@metamask/stake-sdk` peer dependency to `3.0.0` ([#5828](https://github.com/MetaMask/core/pull/5828))
```

Comment:
Hmm, `stake-sdk` doesn't seem like a peer dependency, it seems like a regular dependency. So this doesn't seem like a breaking change to me. Also, it seems we've added two new dependencies:

``` suggestion
 - Bump `@metamask/stake-sdk` dependency to `^3.2.0` ([#5828](https://github.com/MetaMask/core/pull/5828))
 - Add `@ethersproject/bignumber` as a dependency ([#5828](https://github.com/MetaMask/core/pull/5828))
 - Add `reselect` as a dependency ([#5828](https://github.com/MetaMask/core/pull/5828))